### PR TITLE
Add tests for get_chunks and change formula for get_chunks

### PR DESCRIPTION
**Pull request by PokhodenkoSA**
See tests. They show how it should work.

### CODE REVIEW

File: sdc/utilities/prange_utils.py [hunk 1]
```diff
@@ -54,36 +54,19 @@ def get_pool_size_impl():
     return get_pool_size_impl
 
 
+@sdc_register_jitable
 def get_chunks(size, pool_size=0):
     if pool_size == 0:
```
Comment (Hardcode84) [inline sdc/utilities/prange_utils.py]: Suggestion - remove this check and always pass `pool_size` explicitly.

Pure functions are good

Comment (PokhodenkoSA) [reply sdc/utilities/prange_utils.py]: Done.

File: sdc/tests/test_prange_utils.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# *****************************************************************************
+# Copyright (c) 2019, Intel Corporation All rights reserved.
```
Comment (densmirn) [inline sdc/tests/test_prange_utils.py]: Please change year in the copyright.

Comment (PokhodenkoSA) [reply sdc/tests/test_prange_utils.py]: Done

File: sdc/utilities/prange_utils.py (hunk 1 above)
Comment (densmirn) [reply sdc/utilities/prange_utils.py]: So now we should explicitly call `get_pool_size()` and pass it to `get_chunks()`. Am I correct?

Comment (PokhodenkoSA) [reply sdc/utilities/prange_utils.py]: Yes

Comment (densmirn) [reply sdc/utilities/prange_utils.py]: Seems it's not so convenient.

Comment (PokhodenkoSA) [reply sdc/utilities/prange_utils.py]: I can introduce function like `parallel_chunks(N)` which will do call `get_chunks(N, get_pool_size())`. @Hardcode84, @densmirn ok?

Comment (densmirn) [reply sdc/utilities/prange_utils.py]: I like the idea.

Comment (PokhodenkoSA) [reply sdc/utilities/prange_utils.py]: Done

**Pull request merged**